src/components/Combobox/Combobox.module.scss

```diff
@@ -3,10 +3,10 @@
 @import "../../styles/states";
 
 .combobox {
+  @include smoothing-text;
```

Comment:
need to fix this before marge (design review)
![Screen Shot 2023-07-19 at 14 45 49](https://github.com/mondaycom/monday-ui-react-core/assets/72390374/8103bd42-9ca1-49fc-9736-1a2fb6a02822)

Reply:
reason: default line height change. 
Added 4 pixel padding fix fix it (and extra 2 for make the sticky header positioned better)